lib/tasks/dev.rake

```diff
@@ -79,6 +79,8 @@ namespace :dev do
 
   desc 'Create in-person enrollments for N random users'
   task random_in_person_users: [:environment, :random_users] do
+    usps_request_delay_ms = (ENV['USPS_REQUEST_DELAY_MS'] || 0).to_i
+    usps_request_delay_ms.freeze
```

Comment:
Calling `.freeze` on a number doesn't do anything, I don't think (because numbers are already immutable)

```ruby
irb(main):001:0> x = 1
=> 1
irb(main):002:0> x.frozen?
=> true
```

Reply:
Done in 4c5ee1edb